test/Templates.Test/Helpers/AspNetProcess.cs

```diff
@@ -8,23 +8,37 @@
 using Microsoft.Extensions.CommandLineUtils;
 using Xunit;
 using Xunit.Abstractions;
+using Microsoft.AspNetCore.Certificates.Generation;
+using System.Threading;
 
 namespace Templates.Test.Helpers
 {
     public class AspNetProcess : IDisposable
     {
         private const string DefaultFramework = "netcoreapp2.0";
         private const string ListeningMessagePrefix = "Now listening on: ";
+        private static int Port = 5000 + new Random(Seed: 0).Next(3000);
```

Comment:
This doesn't look right. Having a fixed seed means it's always going to return the same sequence. It would be equivalent just to hard-code the specific value returned by `new Random(Seed: 0).Next(3000)` (which is 2178 if I recall correctly).

Reply:
Fair enough, I'll remove the seed part (should pick a different seed each time I hope).
I just want the base port to be random to minimize the change of port conflicts